californium-core/src/main/java/org/eclipse/californium/core/network/EndpointManager.java

```diff
@@ -94,6 +95,36 @@ public static EndpointManager getEndpointManager() {
 	private Endpoint default_secure_tpc_endpoint;
 
 	/**
+	 * Get default endpoint for provided scheme.
+	 * 
+	 * @param scheme scheme to select endpoint
+	 * @return the default endpoint for the provided scheme
+
+	 * @see #getDefaultEndpoint()
+	 * @see #getDefaultSecureEndpoint()
+	 * @see #getDefaultTcpEndpoint()
+	 * @see #getDefaultSecureTcpEndpoint()
+	 */
+	public Endpoint getDefaultEndpoint(String scheme) {
```

Comment:
We need to check, how many test are using the "specialized" methods.
I tried such e replace in the "LeshanServer", but it affects a lot of tests.

I will check, how many changes would be required additionally.